### PR TITLE
Timeshift whitelisting and new helper script

### PR DESCRIPTION
**Pull request by mgerstner**
Whitelist new Timeshift actions and also make a first shot at semi-automating addition of new rules with the help of a Python script.

### CODE REVIEW

**Review by maltek — APPROVED** [review]
LGTM.

I'm wondering if we want to add an option to the tool to do a git commit. It's a lot of magic, but could smooth the workflow further. Something like

    --git-new -> git -c commit.verbose=true commit -m $NEW_GROUP -e -o -- profiles/polkit-default-privs.{easy,standard,restrictive}
    --git-amend -> git commit --no-edit --amend -- profiles/polkit-default-privs.{easy,standard,restrictive}

File: tools/add_polkit_action.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    # existing authentication type settings in increasing order of security
+    AUTH_TYPES = ("yes", "auth_self_keep", "auth_self", "auth_admin_keep", "auth_admin", "no")
+    AUTH_CATEGORIES = ("any-user", "inactive-session", "active-session")
+
+    def __init__(self):
+
+        self.m_profile_dir = os.path.realpath(__file__)
+        self.m_profile_dir = os.path.sep.join(self.m_profile_dir.split(os.path.sep)[:-2])
+        self.m_profile_dir = os.path.sep.join((self.m_profile_dir, "profiles"))
+
```
[the remark this answers is not in the document]
Comment (mgerstner) [reply tools/add_polkit_action.py]: Didn't know `pathlib` yet. I'll have a look.

**Comment by mgerstner** [comment]
> LGTM.
> 
> I'm wondering if we want to add an option to the tool to do a git commit. It's a lot of magic, but could smooth the workflow further. Something like
> 
> ```
> --git-new -> git -c commit.verbose=true commit -m $NEW_GROUP -e -o -- profiles/polkit-default-privs.{easy,standard,restrictive}
> --git-amend -> git commit --no-edit --amend -- profiles/polkit-default-privs.{easy,standard,restrictive}
> ```

I'd leave that for a follow-up PR. I'm also not sure if too much automation is a good thing here. IMO people should carefully inspect what the tool did. Also when we want to add follow-up entries in existing "related action groups" then manual post-editing might be required. If we want to keep this grouping at all that is ...

**Comment by mgerstner** [comment]
I rebased the branch to use the pathlib which looks way nicer now.

**Pull request merged**